bucket/nvidia-display-driver-np.json

```diff
@@ -23,8 +23,8 @@
     ],
     "architecture": {
         "64bit": {
-            "url": "https://us.download.nvidia.com/Windows/472.12/472.12-desktop-win10-win11-64bit-international-whql.exe#/dl.7z_",
-            "hash": "95d8dc470e8f1352b4dbc3b38aefeb5cbbba024826a0541a9f88501b642d81d3"
+            "url": "https://us.download.nvidia.com/Windows/528.24/528.24-desktop-win10-win11-64bit-international-dch-whql.exe#/dl.7z_",
```

Comment:
The addition of `dch` to this URL is a bug which causes us to install the wrong driver version!

I need to remove `dch` from this URL and update the hash, but _also_ change the auto-update logic to look for the new URL

Reply:
It looks like NVIDIA has stopped shipping "Standard" drivers? Maybe this is why the package broke in the first place.

Either way, I changed the auto-update logic so the "standard" package will only install the standard driver